app/src/main/java/com/kamron/pogoiv/widgets/PlayerTeamAdapter.java

```diff
@@ -0,0 +1,62 @@
+package com.kamron.pogoiv.widgets;
+
+import android.content.Context;
+import android.graphics.Color;
+import android.util.TypedValue;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.SpinnerAdapter;
+import android.widget.TextView;
+
+import com.kamron.pogoiv.R;
+
+import java.util.ArrayList;
+
+/**
+ * Created by Johan on 2016-09-06.
+ * An adapter class that makes the teams color coded
+ */
+public class PlayerTeamAdapter extends BaseAdapter implements SpinnerAdapter {
+
+    ArrayList<Integer> colors;
+    Context context;
```

Comment:
Can we keep these private? I know we haven't cared but we've been getting better—I'd like new code to try.